poki-sdk/manifests/web/engine_template.html

```diff
@@ -7,6 +7,17 @@
 
 <body>
 	<script id='poki-sdk-setup' type='text/javascript'>
+		var isWasmLoaded = false;
```

Comment:
Moved all non-related to Poki stuff outside of defer script callback. It alters loading process to wait Poki SDK initialization before full complete poki sdk loading.